cmd/crc/cmd/status_test.go

```diff
@@ -52,7 +53,7 @@ func TestJsonStatus(t *testing.T) {
   "cacheDir": "%s"
 }
 `
-	assert.Equal(t, fmt.Sprintf(expected, cacheDir), out.String())
+	assert.Equal(t, fmt.Sprintf(expected, strings.Replace(cacheDir, `\`, `\\`, -1)), out.String())
```

Comment:
@guillaumerose I am not sure what you are accomplice here, do you want to replace `\` by `\\` in windows so that `cacheDir` should take only single `\` when `fmt.Sprintf` executed? https://play.golang.org/p/-XJ9XdtqjwS

Reply:
In order to create valid JSON string, I need to escape backslashes.